src/main/resources/avro/common.avdl

```diff
@@ -53,293 +54,201 @@ forward orientation. The right side is indicated using the "-" orientation. For
+contiguous pieces of the "ref" `Sequence`, and one of which describes the newly
+added "nov1" `Sequence`:
+
+    1. Segment with top strand GG in the "ref" reference sequence:
+    start = (0,+), "ref"
+    length = 2
+
+    2. Segment with top strand C in the "nov1" reference sequence:
+    start = (0,+), "nov1"
+    length = 1
+
+    3. Segment with top strand TG in the "ref" reference sequence:
+    start = (3,-), "ref"
+    length = 2
+
+    5. Segment with top strand GNG in the "ref" reference sequence:
+    start = (4,+)
+    length = 3
 
 It looks like this:
 
-```
-      --C>-
-     ^     |
-=G>=G=<T=<G==G>=N>=G>
-      |     ^
-       -->--
-```
-
-Notice that we always maintain the coordinate system on the reference sequence.
-The inverted middle segment is specified as starting at a position corresponding
-to a nick in the bottom strand of the reference sequence, and continuing on 2
-basepairs in a right-to-left direction along the reference sequence. It does not
-get new coordinates relative to its final orientation or position in the
-rearranged conformation. Also, all of the novel adjacencies in the sequence
-graph involve actual novel segments, even if some of these are empty. This is
-good for bookkeeping; we want to be able to keep the parent sequences constant
-and add new child sequences to express new variants. Overall, this scheme allows
-us to create sequence graphs describing any number of configurations and
-reconfigurations of reference DNA sequences, and to describe paths through those
-graphs representing particular arrangements.
+          --C>-
+         ^     |
+    =G>=G=<T=<G==G>=N>=G>
+          |     ^
+           -->--
+
+Notice that we always maintain the coordinate system on the reference
+`Sequence`. The inverted middle `Segment` is specified as starting at a position
+corresponding to a nick in the bottom strand of the reference `Sequence`, and
+continuing on 2 basepairs in a right-to-left direction along the reference
+`Sequence`. It does not get new coordinates relative to its final orientation or
+position in the rearranged conformation. This is good for bookkeeping; we want
+to be able to keep the parent `Sequence`s constant and add new child `Sequence`s
+to express new variants. Overall, this scheme allows us to create sequence
+graphs describing any number of configurations and reconfigurations of reference
+DNA sequences, and to describe paths through those graphs representing
+particular arrangements.
 */
 protocol Common {
 
 /**
 Indicates the DNA strand associate for some data item.
-* `POS_STRAND`:  The postive (+) strand.
 * `NEG_STRAND`: The negative (-) strand.
-* `NO_STRAND`: Strand-independent data or data where the strand can not be determined.
+* `POS_STRAND`:  The postive (+) strand.
+Richard and Adam removed NO_STRAND. Instead have a union with null if you want
+to support unstranded annotations.
 */
 enum Strand {
-  POS_STRAND,
   NEG_STRAND,
-  NO_STRAND
```

Comment:
Frank Austin Nothaft <notifications@github.com> writes:
> In src/main/resources/avro/common.avdl:
> 
> >    NEG_STRAND,
> > -  NO_STRAND
> 
> +1 @jeromekelleher

-1, NO_STRAND  is required for genome annotations.

Although it would be perfectly reasonable to have two different
stand representation, I would +1 that if it's *really* need and we
create two enums where POS and NEG have the same value.

But it makes it a lot easier if there isn't.